Carthage/Checkouts/stackviewcontroller/README.md

```diff
@@ -1,5 +1,4 @@
-# StackViewController
-[![Circle CI](https://circleci.com/gh/seedco/StackViewController/tree/master.svg?style=svg)](https://circleci.com/gh/seedco/StackViewController/tree/master)
```

Comment:
Why remove the Circle CI badge?

Reply:
Here's the commit that changed that: https://github.com/seedco/StackViewController/commit/6649a15141c397d625b842f384351e0fdc547f34#diff-04c6e90faac2675aa89e2176d2eec7d8